examples/mobile/package.json

```diff
@@ -4,7 +4,8 @@
     "private": true,
     "scripts": {
         "start": "node node_modules/react-native/local-cli/cli.js start",
-        "test": "jest"
+        "test": "jest",
+        "postinstall": "cd node_modules/react-native; exec ./scripts/ios-install-third-party.sh; cd third-party/glog-0.3.5; exec ./configure; cd ../../../.."
```

Comment:
Temporary hotfix for https://github.com/facebook/react-native/issues/20774. The root cause should be fixed in RN 0.57.5